src/Core/Domain/Product/ValueObject/ProductId.php

```diff
@@ -29,7 +29,7 @@
 use PrestaShop\PrestaShop\Core\Domain\Order\Exception\OrderException;
 
 /**
- * Product identity
+ * Product identity.
```

Comment:
Nice :trollface:

Reply:
Haha, this happened after rebase. :smile: